os/board/rtl8721csm/src/component/common/bluetooth/realtek/sdk/example/da_ble_peripheral/ble_tizenrt_app.c

```diff
@@ -190,19 +190,19 @@ void ble_tizenrt_app_handle_io_msg(T_IO_MSG io_msg)
 }
 
 extern da_ble_server_init_parm server_init_parm;
-void ble_tizenrt_handle_callback_msg(T_TIZENRT_SERVER_CALLBACK_MSG callback_msg)
+void ble_tizenrt_handle_callback_msg(T_TIZENRT_APP_CALLBACK_MSG callback_msg)
 {
     debug_print("\r\n[%s] msg type: 0x%x", __FUNCTION__, callback_msg.type);
     switch (callback_msg.type) {
         case BLE_TIZENRT_CALLBACK_TYPE_CONN:
         {
-            T_TIZENRT_CONNECTED_CALLBACK_DATA *connected = callback_msg.msg_data;
+            T_TIZENRT_CONNECTED_CALLBACK_DATA *connected = callback_msg.u.buf;
             debug_print("\r\n[%s] cb %p conn_id 0x%x conn_type 0x%x addr 0x%x0x%x0x%x0x%x0x%x0x%x", __FUNCTION__,
                             server_init_parm.connected_cb,
                             connected->conn_id, connected->conn_type,
                             connected->remote_bd[0], connected->remote_bd[1], connected->remote_bd[2],
                             connected->remote_bd[3], connected->remote_bd[4], connected->remote_bd[5]);
-            if(server_init_parm.connected_cb)
+            if(connected != NULL && server_init_parm.connected_cb)
```

Comment:
`connected` is also used from line 220(in `debug_print`). So I think null checking should be before line 220.

Reply:
Fixed.